VideoLocker/src/main/java/org/edx/mobile/view/custom/ECheckedTextView.java

```diff
@@ -0,0 +1,40 @@
+package org.edx.mobile.view.custom;
+
+import android.content.Context;
+import android.content.res.TypedArray;
+import android.graphics.Typeface;
+import android.util.AttributeSet;
+import android.widget.CheckedTextView;
+
+import org.edx.mobile.R;
+
+public class ECheckedTextView extends CheckedTextView {
+    public ECheckedTextView(Context context) {
+        super(context);
+    }
+
+    public ECheckedTextView(Context context, AttributeSet attrs) {
+        super(context, attrs);
+        processAttrs(context, attrs);
+    }
+
+    public ECheckedTextView(Context context, AttributeSet attrs, int defStyleAttr) {
+        super(context, attrs, defStyleAttr);
+        processAttrs(context, attrs);
+    }
+
+    private void processAttrs(Context context, AttributeSet attrs) {
+        if (isInEditMode()) return;
+        TypedArray a = context.getTheme().obtainStyledAttributes(
+                attrs, R.styleable.custom_view, 0, 0);
+        // Check for the font attribute and setup font
+        String fontFileName = a.getString(R.styleable.custom_view_font);
+        a.recycle();
+        if (fontFileName == null) {
+            // If font is not defined, then set default font
+            fontFileName = "OpenSans-Regular.ttf";
```

Comment:
@miankhalid: We can simply define the `textAppearance*` theme attributes to set the default font for all widgets using Calligraphy (and probably the data binding library), with the added bonus of complete customizability.